scrapers/covid-19/nytData.js

```diff
@@ -27,8 +27,8 @@ const nytData = async (keys, redis) => {
 			US_STATE_DATA_URL,
 			US_NATION_WIDE_DATA_URL
 		].map(_resolveData));
-		logger.info('NYT Data successfully retrieved');
-		await updateNYTCache();
+		logger.info('NYT Data successfully retrieved and cache updated');
```

Comment:
Why do we need the cache anymore at all? We aren't caching we're just reading from Redis